### PR TITLE
CP-2 Add input component

### PR DESCRIPTION
**Pull request by githubTestUser21**
It add a fancy web component

### CODE REVIEW

File: component/input/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function fancyComponent1() {
+    return 1 +1;
```
Comment (githubTestUser20) [inline component/input/index.js]: I'm not sure about this. Please update.

File: component/input/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+}
```
Comment (githubTestUser20) [inline component/input/index.js]: A missing space here.

File: component/input/index.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+function fancyComponent1() {
```
Comment (githubTestUser20) [inline component/input/index.js]: The method should not have numeric characters

**Review by githubTestUser20 — CHANGES_REQUESTED** [review]

File: component/input/index.js (hunk 3 above)
Comment (githubTestUser21) [reply component/input/index.js]: I'm not agreed 👎

File: component/input/index.js (hunk 1 above)
Comment (githubTestUser21) [reply component/input/index.js]: Yeah, a space is missing, I'm gonna  🔨  it

File: component/input/index.js (hunk 2 above)
Comment (githubTestUser21) [reply component/input/index.js]: Ok 🙇

File: component/input/index.js (hunk 3 above)
Comment (githubTestUser20) [reply component/input/index.js]: one word, code guide lines.

File: component/input/index.js (hunk 2 above)
Comment (githubTestUser20) [reply component/input/index.js]: Function name is to long ! 👎

**Review by githubTestUser20 — CHANGES_REQUESTED** [review]

File: component/input/index.js (hunk 2 above)
Comment (githubTestUser21) [reply component/input/index.js]: ARKM ? 😠

Comment (githubTestUser21) [reply component/input/index.js]: Ok. I'll do it 🉑

File: component/input/index.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-function fancyComponentOne() {
+function shortName() {
```
Comment (githubTestUser20) [inline component/input/index.js]: Now I like it ! 😃

**Review by githubTestUser20 — APPROVED** [review]

File: component/input/index.js (hunk 4 above)
Comment (githubTestUser21) [reply component/input/index.js]: Finally !! 🎆

Comment (githubTestUser21) [reply component/input/index.js]: @githubTestUser20 can you merge it?

**Pull request merged**